### ISSUE TITLE
Chinese language support

### ISSUE THREAD

**Issue by YouLe2016** [DanteCoder/JoplinBibleQuote#30]
可以支持和合本（神版）圣经吗？使用中文的大多都是用的这个版本

Translation: Can I support the Hutchinson (God's Version) Bible? This is the version used by most Chinese speakers.

**Comment by DanteCoder** [comment]
Yes, I will look into it. I will try to find and upload the bible in osis format to this repo and add support for the chinese language, though for the bible parser (writing the citations) I don't know if there will be support for chinese as I'm depending on an external library.

**Comment by DanteCoder** [comment]
@YouLe2016 Hi, I know it's been a long time, I've been busy.
Can you tell me [if this version](https://drive.google.com/file/d/1m1vT8gJDPa0V3-n9q_l3_hSsNU-tnmYm/view?usp=sharing) is the right one?
You can test it directly in the plugin.

**Comment by elizhyu** [comment]
> @YouLe2016 Hi, I know it's been a long time, I've been busy. Can you tell me [if this version](https://drive.google.com/file/d/1m1vT8gJDPa0V3-n9q_l3_hSsNU-tnmYm/view?usp=sharing) is the right one? You can test it directly in the plugin.

@DanteCoder
I checked the file you provided. It is the **correct** one.
More specifically, it is the `Big5` (`traditional Chinese`) encoded `Chinese Union Version`, while there is another commonly used `Chinese Union Version` encoded by `GB2312` (`simplified Chinese`).

I've got the [text version](https://github.com/DanteCoder/JoplinBibleQuote/files/13470004/hgb.txt) for that available below, if possible, maybe you are able to convert it to the XML OSIS format and added to the repo?

**Comment by DanteCoder** [comment]
# @elizhyu 
Thanks for the clarification!
I'm looking forward to parsing the text version you provided and converting it to OSIS format.

**Comment by DanteCoder** [comment]
@elizhyu 
So I created a little Python script to parse the text file into a format that I could then use on the "Simple Bible Reader" software to convert it into an OSIS file.
I'm not sure if the encoding got broken in the conversion process, as I'm not able to visualize the Chinese characters on my computer, so I'd be grateful if you could tell me if this file works for you: 
[Chinese Union Version.zip](https://github.com/DanteCoder/JoplinBibleQuote/files/13531404/Chinese.Union.Version.zip)

**Comment by elizhyu** [comment]
@DanteCoder 
I tried the OSIS file you converted for `GB` encoding, but it seems corrupted. No matter what encoding I tried to open it with, the verses are showing as meaningless characters.

Hence, I will say for now it might be ok to just use the `Big5` encoding to represent the `Chinese Union Version`, since I, even though as a `Mandarin` speaker who uses `Simplified Chinese` day to day, can still read 99% of the `Traditional Chinese` context.
